recipe/build.sh

```diff
@@ -1,2 +1,6 @@
 #!/bin/bash
+
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```

Comment:
This file doesn't need to be changed in this PR because it was initially created by the helper script. Could you please discard all changes to `build.sh`?